### DISCUSSION TITLE
RenderStream Projects folder

### DISCUSSION THREAD

**Discussion by avanalle** [disguise-one/RenderStream-UE#30] [Q&A] [answered]
Is there a way to point d3 to the appropriate renderstream projects folder? We have a lab of 20 machines and since designer was installed with an image there does not appear to be a renderstreamprojects folder. When I try to choose the asset d3 is pointing to a nonexistent disguise/disguise folder

**Comment by josh-disguise** [comment, marked as answer]
The renderstream projects folder is controlled by a registry key, `HKEY_CURRENT_USER\Software\d3 Technologies\d3 Production Suite\RenderStream Projects Folder` 

![regedit](https://user-images.githubusercontent.com/80043790/132724729-3a47025d-31f7-4ead-83d4-187d16a4fb80.PNG)

You would need to reboot or relaunch d3service after changing it. Does that solve your issue?

**Reply by avanalle** [reply]
Ah ha. I do not even have that registry key listed. I am assuming there is some sort of fault with how things are directed.

**Reply by avanalle** [reply]
or rather how these machines had the images deployed.

**Reply by avanalle** [reply]
Hey Josh. Random unrelated question. I was able to get d3 to locate the UE assets by creating and moving the renderstream projects folder to the D:\. That was extremely helpful so thank you again for pointing me toward killing the service and all that. However now when I go to start the workload I get a notification telling me the system cannot find the path specified. not sure what I'm missing here
![image](https://user-images.githubusercontent.com/90408115/132744480-3cda4ea8-d80c-44f0-afdf-9335a1a9544e.png)

**Reply by josh-disguise** [reply]
Not sure, @husberg-disguise do you know?

**Reply by husberg-disguise** [reply]
It looks like it's trying to create a folder called "". That function is trying to iteratively create a folder structure by splitting the path into parts separated by slashes, so make sure you don't have and extra slash somewhere in the RenderStream Projects path.
